recipe/0003-jpeg8_use_shared_libjpeg.patch

```diff
@@ -1,12 +1,13 @@
 diff --git a/setup.py b/setup.py
+index a7568a1..5e00997 100644
 --- a/setup.py
 +++ b/setup.py
-@@ -145,7 +145,7 @@ EXTENSIONS = {
-         libraries=['openjp2', 'lcms2'],
+@@ -180,7 +180,7 @@ EXTENSIONS = {
      ),
      'jpeg8': ext(
+         sources=['imagecodecs/_jpeg8_legacy.pyx'],

```

Comment:
I'm not sure this patch is needed any more, see [here](https://github.com/AnacondaRecipes/jpeg-feedstock/pull/6) and [here](https://github.com/AnacondaRecipes/jpeg-feedstock/pull/7)

Reply:
I removed this patch. Thanks!